src/fides/api/ops/models/messaging.py

```diff
@@ -127,7 +127,7 @@ def get_by_type(
         """
         Retrieve the messaging config of the given type
         """
-        return db.query(cls).filter_by(service_type=service_type).first()
+        return db.query(cls).filter_by(service_type=service_type.value).first()
```

Comment:
`sqlalchemy` must be auto-translating enums in the background for us for this to have worked originally?